libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -845,9 +845,15 @@ AP_InertialSensor::init(uint16_t loop_rate)
     }
 
     // calibrate gyros unless gyro calibration has been disabled
-    if (gyro_calibration_timing() != GYRO_CAL_NEVER) {
+    switch (gyro_calibration_timing()) {
+    case GYRO_CAL_NEVER:
+        break;
+    case GYRO_CAL_STARTUP_ONLY:
         init_gyro();
-    }
+        break;
+    default:
```

Comment:
Do you want a default? If we add a new option to the enum then not having something for the switch should trigger compiler errors and help us implement it... (Also not sure that having a crazy value here will be all that terrible (not great, just not all that awful?)

Reply:
The idea was to get people to fix their parameters so we could add new values to the enumeration without feat we'd do random things in patches which ascribed new meanings to values.

Well, we decided to do the latter rather than burning bytes here - we're closing this PR.

But you're also right - I shouldn't have had the default, rather had some other mechanism to catch the bad values (even a boolean value or whatnot)